g_docs.go

```diff
@@ -1291,3 +1287,14 @@ func getPackageName() (string, error) {
 
 	return strings.ReplaceAll(pwd, gopathSRC, ""), nil
 }
+
+// getProjectFromImportPath extracts the project name from the import path.
+// Assumption: package path has always the form github.com/<user>/<project>
+func getProjectFromImportPath(path string) (string, error) {
+	pathParts := strings.Split(path, string(os.PathSeparator))
+	if len(pathParts) < 3 {
+		return "", errors.New("unrecognized import path form")
```

Comment:
nit: do you want to add the given path to the error message? Although I see it's not used anyway 😅